boxoffice/views/utils.py

```diff
@@ -22,9 +22,14 @@ def localize(datetime, tz):
     return utc.localize(datetime).astimezone(timezone(tz))
 
 
-@app.template_filter('invoice_date')
-def invoice_date_filter(date, format):
-    return localize(date, app.config['TIMEZONE']).strftime(format)
+@app.template_filter('date_time_format')
+def date_time_format(datetime):
+    return utc.localize(datetime).astimezone(timezone(app.config['TIMEZONE'])).strftime('%d %b %Y %H:%M:%S')
+
+
+@app.template_filter('date_format')
+def date_format(datetime):
+    return utc.localize(datetime).astimezone(timezone(app.config['TIMEZONE'])).strftime('%d %b %Y')
```

Comment:
This code is a bit muddled.

1. [Baseframe inserts `app.config['tz'] = timezone(app.config.get('TIMEZONE', 'UTC'))` for you](https://github.com/hasgeek/baseframe/blob/8e83e0109bcec9e8f18842e59d3d63e9a576c019/baseframe/__init__.py#L175). You can just use `app.config['tz']` with the assurance it exists.
2. If the user is logged in (`g.user` is not None), you need to use `g.user.tz` ([which is also guaranteed to be `timezone(g.user.timezone or app.config['TIMEZONE'])`](https://github.com/hasgeek/flask-lastuser/blob/master/flask_lastuser/sqlalchemy.py#L130-L141)).

To reconcile these two, I've added a new function to baseframe that can you can use: [localize_timezone](https://github.com/hasgeek/baseframe/blob/63a3ef6d4c91605d1dfe0dce12b4436a8b876139/baseframe/__init__.py#L236-L243)